pom.xml

```diff
@@ -1122,6 +1043,12 @@
               <artifactId>jaxb-impl</artifactId>
               <version>${jaxb-impl.version}</version>
             </dependency>
+            <dependency>
+              <groupId>com.sun.xml.ws</groupId>
```

Comment:
This dependency seems to bring in all of the JAX-WS and JAXB dependencies Rice needs which were removed in Java 11.